src/app/conf/2024/sponsor/index.mdx

```diff
@@ -2,8 +2,10 @@ import { Button } from '@/app/conf/_components/button'
 
 # Sponsor GraphQLConf 2024
 
-<Button href="mailto:graphqlconf@graphql.org?subject=Sponsorships" className="!text-white mx-auto !block">Contact Us</Button>
-
+<div className="flex flex-row justfiy-center">
+    <Button target="_blank" href="/conf/sponsor/sponsor_2024.pdf" className="!text-white !block">Sponsership Details</Button>
```

Comment:
@Urigo you can suggest a change and commit it here if you want a different label for the button